### PR TITLE
Ad API to unobserve intersections from 3p frame

### PR DESCRIPTION
**Pull request by sriramkrish85**
#1340

### CODE REVIEW

**Comment by ampsauce** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: 5c65a2c2ae28d87d9d33a8ba9c3e7714e0d06418
Build details: https://travis-ci.org/ampsauce/amphtml/builds/104198976

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by ampsauce** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: ffc469b3eef67bbd73d8f523051e6a9463a3d920
Build details: https://travis-ci.org/ampsauce/amphtml/builds/104199428

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

File: 3p/integration.js [hunk 1]
```diff
@@ -166,11 +166,11 @@ function nonSensitiveDataPostMessage(type, opt_object) {
  * the IntersectionObserver spec callback.
  * http://rawgit.com/slightlyoff/IntersectionObserver/master/index.html#callbackdef-intersectionobservercallback
  * @param {function(!Array<IntersectionObserverEntry>)} observerCallback
+ * @returns {!function} A functions which removes the event listener that
```
Comment (cramforce) [inline 3p/integration.js]: nit: function singular

**Comment by cramforce** [comment]
Document in `ads/README.md`

**Comment by ampsauce** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: a6dc0dba3aa924505b3a687c60551ebfed3c6f4b
Build details: https://travis-ci.org/ampsauce/amphtml/builds/104219973

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

File: 3p/integration.js (hunk 1 above)
Comment (sriramkrish85) [inline 3p/integration.js]: Done!

**Comment by sriramkrish85** [comment]
PTAL

**Comment by ampsauce** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: cc251de1096ac661490c9ae9079a87c22ed4ab12
Build details: https://travis-ci.org/ampsauce/amphtml/builds/104223832

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by cramforce** [comment]
LGTM

**Pull request merged**